src/components/button/examples/button-click-success.tsx

```diff
@@ -3,13 +3,21 @@ import { Component, h, State } from '@stencil/core';
 /**
  * With click handler
  *
- * The click handler in this example sets the attributes `loading` and
- * `disabled` to `true`. After 1 second, the `loading` attribute is set to
- * `false` again. After another 5 seconds, the button is once again enabled.
+ * The click handler in this example simulates saving some changed values in a
```

Comment:
I also updated this example description, so that it matches the new functionality, but also so that the intention behind the example is hopefully clearer now than before.

Reply:
@Kiarokh Tell me if you think my changes adequately explains the example, and how the button is intended to be used in reality, or if you think there's anything that's still in need of improvement.

Reply:
Looks great Adrian!